.jshintrc

```diff
@@ -20,5 +20,7 @@
 		"Backbone": false,
 		"jQuery": false,
 		"wp": false
-	}
+	},
+
+    "predef": [ "wpsc_vars" ]
```

Comment:
It's definitely not an edge case.  If we put a global in the .js file, and it is also always declared in the CDATA within the HTML, a global is being declared at least two times.

I feel strongly enough that this would be a such a bad practice I won't make the change as I don't want my name associated with it should it blow up.